quodlibet/quodlibet/qltk/tracker.py

```diff
@@ -69,7 +82,7 @@ def __paused(self, player, paused):
         else:
             self.__stop = False
             if self.__id is None:
-                self.__id = GLib.timeout_add_seconds(1, self.__update)
+                self.__id = GLib.timeout_add(self.__interval, self.__update)
```

Comment:
TimeTracker uses timeout_add_seconds() because that gets synced with all other timeout_add_seconds() calls (also from other apps) and allows us to wake up only once per second. 

So in case it is 1000 it should still call timeout_add_seconds().